src/com/lftechnology/java/training/srijan/basics/football/ManchesterUnited.java

```diff
@@ -0,0 +1,70 @@
+package com.lftechnology.java.training.srijan.basics.football;
+
+import java.util.logging.Logger;
+import java.util.InputMismatchException;
+import java.util.NoSuchElementException;
+import java.util.Scanner;
+
+public class ManchesterUnited {
+
+	private static final Logger LOGGER = Logger.getLogger(ManchesterUnited.class.getName());
+	
+	public static void main(String[] args) {
+		int numberOfGameWeeks = 0;
+		String club = "";
+		Scanner input = new Scanner(System.in);
+		Scanner clubNameString = new Scanner(System.in);
+		try{
+			LOGGER.info("Enter the number of GameWeek");
+			numberOfGameWeeks = input.nextInt();
+			LOGGER.info("Enter club's name your want to see stat of:");
+			club = clubNameString.nextLine();
+			inputFixtures(numberOfGameWeeks, club);
+		}catch(InputMismatchException ex){
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)